src/lib.rs

```diff
@@ -40,8 +40,13 @@
 )]
 #![allow(clippy::match_bool)]
 #![forbid(unsafe_code)]
-#![cfg_attr(all(feature = "nightly", doctest), feature(external_doc))]
-#![cfg_attr(all(docsrs, feature = "nightly"), feature(doc_cfg))]
+// we pass "--cfg docsrs" when building docs to add `This is supported on feature="..." only.`
+//
+// To properly build docs of this crate run
+// ```console
+// $ RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --open --all-features
+// ```
+#![cfg_attr(all(docsrs, feature="nightly"), feature(external_doc, doc_cfg))]
```

Comment:
That's not right, `external_doc` feature should stay with `doctest`, not `docsrs`.

Reply:
:man_facepalming: stupid me... I though it says `docsrs` :sweat_smile: